### ISSUE TITLE
Rear2.3 asking for :genisoimage for RHEL 5.8

### ISSUE THREAD

**Issue by nirmal21s** [rear/rear#1837]
#### Relax-and-Recover (ReaR) Issue Template

Fill in the following items before submitting a new issue
(quick response is not guaranteed with free support):

* ReaR version ("/usr/sbin/rear -V"): 2.3
* OS version ("cat /etc/rear/os.conf" or "lsb_release -a" or "cat /etc/os-release"):RHEL 5.8
Arch : i686

* Brief description of the issue:
When trying to install the REAR 2.3 rpm for RHEL 5 its not getting installed .
It requires Genisoimage.
There is no package genisoimage available for RH5.
Is there any workaround available to make it work.

[geniso_error.txt](https://github.com/rear/rear/files/2122968/geniso_error.txt)

excerpts:
<pre>
yum install rear-2.3-185.git.0.d642b65.unknown.el5.i386.rpm 
...
Error: Missing Dependency: genisoimage is needed by package rear-2.3-185.git.0.d642b65.unknown.el5.i386 (/rear-2.3-185.git.0.d642b65.unknown.el5.i386)
 You could try using --skip-broken to work around the problem
</pre>

**Comment by jsmeix** [comment]
I am not a RHEL user but I would ignore that yum error
(i.e. force yum to install it nevertheless).

What you ned is a 'genisoimage' compatible program
on your RHEL system installed but it does not matter
from which RPM package name it comes.

A 'genisoimage' compatible program is ususually called
during "rear mkrescue/mkbackup" when the ReaR recovery system
ISO image is made (i.e in case of `OUTPUT=ISO`).

You can specify your particular  'genisoimage' compatible program
via ISO_MKISOFS_BIN (see its description in default.conf).


that is neded

**Comment by jsmeix** [comment]
Since
https://github.com/rear/rear/issues/1192
and its
https://github.com/rear/rear/commit/a3bf4b6e08272f6247e5e8d821c53e9b51b9a8cd
there is in
https://github.com/rear/rear/blob/master/packaging/rpm/rear.spec
<pre>
### On RHEL/Fedora the genisoimage packages provides mkisofs
%if %{?centos_version:1}%{?fedora:1}%{?rhel_version:1}0
Requires: crontabs
Requires: iproute
#Requires: mkisofs
Requires: genisoimage
#Requires: redhat-lsb
%endif
</pre>
which is probably incompatible with RHEL5.

**Comment by jsmeix** [comment]
@nirmal21s 
what is the RPM package name on RHEL5
that provides the 'genisoimage' compatible program?

From the above rear.spec excerpt I guess in RHEL5
the 'genisoimage' compatible program is 'mkisofs'
and its RPM package name is also 'mkisofs'
but we need to be sure what there actually is on RHEL5
(I assume no ReaR developer still uses that somewhat old system).

**Comment by gdha** [comment]
A last comment about RHEL5 - it is unsupported by RedHat since March 2017, so do not expect any fixes in our rear.spec file for it anymore. You can still use the latest rear version, but you need to install it via the work-around given above.
Therefore, I remove the label 'minor bug' and replace it with 'support/question'

**Comment by jsmeix** [comment]
@gdha 
but ReaR still supports RHEL5 because current
https://github.com/rear/rear/blob/master/doc/rear-release-notes.txt
reads (excerpts):
<pre>
ReaR-2.4 is supported on the following Linux based operating systems:
...
* RHEL 5, 6 and 7
...
ReaR-2.4 dropped officially support for the following Linux based operating systems:
...
* RHEL 3 and 4
</pre>
Perhaps we should drop official support for RHEL 5 in Rear 2.4?

**Comment by gdha** [comment]
@jsmeix We will kick it out in ReaR 2.5 as it is still usable with the work-around to given above. We can add it in the release notes (as an add-on).

**Comment by jsmeix** [comment]
@gdha 
anything about RHEL is o.k. for me
(because I am not a RHEL user ;-)

FYI:
I had dropped official support for SLES 10 ReaR 1.17
regardless that even current ReaR should still work with it
because I do not want to get "officially blamed" in any way
if by some change ReaR may no longer work with SLES10, cf.
https://github.com/rear/rear/pull/1765#issuecomment-381520313

**Comment by jsmeix** [comment]
I assume with
https://github.com/rear/rear/issues/1837#issuecomment-399076717
this issue can be considered as sufficiently "fixed/solved/done".

**Comment by nirmal21s** [comment]
@gdha 
I tried this option "rpm -Uvh --nodeps rear-2.3-185.git.0.d642b65.unknown.el5.i386.rpm"
It got installed and created ISO image.
But not yet tested.
Hope this works!
